packages/frontend/src/features/omnibar/components/Omnibar.tsx

```diff
@@ -180,12 +178,11 @@ const Omnibar: FC<Props> = ({ projectUuid }) => {
                             onChange={(e) => setQuery(e.currentTarget.value)}
                         />
 
-                        {hasValidSearchQuery && (
-                            <OmnibarFilters
-                                searchFilter={searchFilter}
-                                onSearchFilterChange={setSearchFilter}
-                            />
-                        )}
+                        <OmnibarFilters
+                            onSearchFilterChange={(filters) => {
+                                setSearchFilters(filters);
+                            }}
+                        />
```

Comment:
I think this makes more sense.. I was trying to keep the UI clean and uncluttered but its way better to show the filters

Reply:
Also, you can make your first query with the filters right away which I think is nice